### DISCUSSION TITLE
Images problem on Simulator

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#214] [Developer Feedback]
Hello.

I'm using Simulator 1.2.1 on linux with target GTR4. Roms 1.0.3 and 1.0.4 can't display images in simulator, but preview on real device works correctly. 
Is is intended behavior or program need some adjustments ?

**Comment by MyoungXUE** [comment]
Hi, thanks for your feedback, we will troubleshoot the simulator as soon as possible!

**Comment by MyoungXUE** [comment]
@kartun83 
Hi, I didn't reproduce the problem, hopefully you can provide more clues, such as a screenshot of the simulator where the image doesn't show up.

**Reply by kartun83** [reply]
Hello. I don't know how to demonstrate it with screenshots.
[img_demo.zip](https://github.com/zepp-health/zeppos-forum/files/12093410/img_demo.zip)
![photo_2023-07-19_12-32-42](https://github.com/zepp-health/zeppos-forum/assets/5894573/ed0c4a5b-648e-4439-a6e8-8f274e805872)

**Reply by MyoungXUE** [reply]
Thank you very much for the project source code!

Some of the images use formats that are not friendly to Zepp OS and cannot be displayed, we suggest you refer to the IMG documentation to adjust the image format.

<img width="652" alt="image" src="https://github.com/zepp-health/zeppos-forum/assets/107526244/e015018c-f9ad-4a25-b9a0-2e19bd4c1f41">

https://docs.zepp.com/docs/reference/device-app-api/newAPI/ui/widget/IMG/

**Reply by kartun83** [reply]
Thanks.
Would be nice if simulator or building tools would be more verbose about such problems.
By the way, if real device could handle such images why simulator can't ?
Additionally - this project was made using watchface maker, and it was able to handle this images whiteout any complains.

**Reply by MyoungXUE** [reply]
Very good suggestion, we've updated this in the beta version of the Zeus-CLI tool to prompt for an image format error when running the zeus dev command.

```js
npm i @zeppos/zeus-cli@1.3.0-beta.10
```

When running on the real machine, we do a conversion of the image, whereas in the simulator preview the Mini Program, there is no such step, resulting in unsupported image formats not being displayed correctly on the simulator.
